lib/mdot/client.rb

```diff
@@ -132,5 +132,11 @@ def handle_error(error)
         raise error
       end
     end
+
+    def handle_token
+      existing_token = MDOT::Token.find(@user.uuid)
+      get_supplies if !existing_token || existing_token.ttl < 5
```

Comment:
The [current ttl setting](https://github.com/department-of-veterans-affairs/vets-api/blob/master/config/redis.yml#L67) looks to be 30 minutes, for future reference.